api/src/main/resources/apps/authenticationui_extension.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
This configures the authenticationui module with appropriate values for Malawi distribution, tying it into the existing user interface and legacyui pages.